ui/component/core/src/index.ts

```diff
@@ -69,7 +69,9 @@ export enum OREvent {
     EVENTS_DISCONNECTED = "EVENTS_DISCONNECTED",
```

Comment:
As mentioned in my comment `ONLINE/OFFLINE` events would be more appropriate and EVENTS_DISCONNECTED/EVENTS_CONNECTED could be replaced with `ONLINE/OFFLILNE` also

Reply:
Can always console log what exactly fails (manager or keycloak)